site/_resources/upgrading.md

```diff
@@ -5,126 +5,122 @@ layout: page
-```
-
-### Recommended Envoy version
-
-The recommended version of Envoy remains unchanged.
-Ensure the Envoy image version is `docker.io/envoyproxy/envoy:v1.11.2`.
-
-## Upgrading Contour 1.0.0-beta.1 to 1.0.0-rc.1
-
-Contour 1.0.0-rc.1 moves HTTPProxy to v1 and continues polishing the release towards the Contour 1.0 release.
-
-### Release candidate
-
-Contour 0.15.3 remains the current stable release.
-The `:latest` tag will continue to point to 0.15.3 until Contour 1.0.0 is released.
 
-### HTTPProxy v1
+This will remove both the Envoy and Contour pods from your cluster
+and recreate them with the updated configuration. If you're using
+a `LoadBalancer` Service, deleting and recreating may change the
+public IP assigned by your cloud provider.  You'll need to re-check
+where your DNS names are pointing as well, using
+[Get your hostname or IP address]({% link _guides/deploy-options.md %}#get_your_hostname_or_ip_address).
 
-Contour 1.0.0-rc.1 promotes the HTTPProxy CRD to v1.
-HTTPProxy is now considered stable, there will only be additive changes in the future.
+## The less easy way
 
-See the [HTTPProxy documentation]({% link _docs_1_0/httpproxy.md %}) for more information.
+This section contains information for administrators who wish to
+manually upgrade from Contour 1.5.3 to Contour 1.0.0.
 
-### `contour.heptio.com` annotations deprecated
+### Upgrade to Contour 1.0.0
 
-All the annotations with the prefix `contour.heptio.com` have been migrated to their respective `projectcontour.io` counterparts.
-The deprecated `contour.heptio.com` annotations will be recognized through the Contour 1.0 release, but are planned to be removed after Contour 1.0.
+Change the Contour image version to `docker.io/projectcontour/contour:v1.0.0`.
 
-See the [annotation documentation]({% link _docs_1_0/annotations.md %}) for more information.
+Note that as part of sunsetting the Heptio brand, Contour Docker
+images have moved from `gcr.io/heptio-images` to
+`docker.io/projectcontour`.
 
-## The easy way to upgrade
+### Reapply HTTPProxy and IngressRoute CRD definitions
 
-If the following are true for you:
+Contour 1.0.0 ships with updated OpenAPIv3 validation schemas.
 
- * Your previous installation is in the `projectcontour` namespace.
- * You are using one of the [example]({{ site.github.repository_url }}/blob/v1.0.0-rc.1/examples/) deployments.
- * Your cluster can take few minutes of downtime.
+Contour 1.0.0 promotes the HTTPProxy CRD to v1. HTTPProxy is now
+considered stable, and there will only be additive, compatible
+changes in the future. See the [HTTPProxy documentation]({% link
+_docs_1_0/httpproxy.md %}) for more information.
 
-Then the simplest way to upgrade to 1.0.0-rc.1 is to delete the `projectcontour` namespace and reapply the `examples/contour` sample manifest.
-From the root directory of the repository:
 ```
-kubectl delete namespace projectcontour
-
-kubectl apply -f examples/contour
+$ kubectl apply -f examples/contour/01-crds.yaml
```

Comment:
I don't think this is sufficient when going from 0.15 to 1.0. @youngnick, thoughts?

Reply:
Per Slack, @youngnick thinks this is enough. I can test this if you point me in the right direction ...

Reply:
Can you check that v1alpha1 HTTPProxy is gone from `kubectl get crd`? I think it should be, as we change the definition and remove v1alpha1 from the supported versions. I can't see it in my cluster, but can't remember if I needed to do anything special for that when I was fiddling with versions.

Reply:
We will need to have instructions on removing the old CRDs when we completely remove them, but that's for later.